third_party/2and3/click/_termui_impl.pyi

```diff
@@ -10,3 +10,5 @@ class ProgressBar(object, Generic[_T]):
     def __iter__(self) -> ProgressBar[_T]: ...
     def next(self) -> _T: ...
     def __next__(self) -> _T: ...
+    length: Optional[int]
+    label: Optional[str]
```

Comment:
The implementation actually ensures that `label` is not `None`, so it should not be `Optional`:

https://github.com/pallets/click/blob/c6042bf2607c5be22b1efef2e42a94ffd281434c/click/_termui_impl.py#L68

Reply:
Ah, you're right... I pushed a follow-up commit just now.